test/framefusion.test.ts

```diff
@@ -368,14 +373,15 @@ describe('FrameFusion', () => {
             1.5, // forward to 46
             0.5, // backward to 16
         ];
+        const rgbaFrameBuffer = new Uint8ClampedArray(extractor.width * extractor.height * 4);
```

Comment:
`4` because it is rgba (4 channel)? consider adding a const variable either at the top of the test or as part of the actual code.

Reply:
Fixed